apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/CrashActivity.java

```diff
@@ -158,6 +159,18 @@ public void run() {
                 public void run() {
                     nativeAbortCall();
                 }
+            }),
+            new Crash(R.string.title_low_memory_warning, R.string.description_oom, new Runnable() {
+
+                @SuppressWarnings({"MismatchedQueryAndUpdateOfCollection", "InfiniteLoopStatement"})
+                @Override
+                public void run() {
+                    int value = 5;
```

Comment:
This is not really different from the existing oom crash buttons.
Basically I tried on emulator and cannot make this work, I never get a warning.
@jaelim-ms how did you test?

Reply:
What I did was similar but my list contained an array of int.
```
public void run() {
  sList = new ArrayList<>();
  for (int i = 0; i < 10000000; i++) {
    sList.add(new Integer[10]);
  }
}
```
Most important thing is that the list should NOT be a local variable since it could be GCed to reserve additional space.

Reply:
Let's update the test and make sure it's tested in the app successfully then.

Reply:
I had a warning when I tested, although I tested just on app, not the SDK API. For some reason the warning handler does get called only if it is overriden in the Application. I will do some more testing now

Reply:
Umfff... So finally with the help of NDK works.